This is a testharness.js-based test.
[FAIL] Blocked eval in DedicatedWorkerGlobalScope.
  assert_throws_js: function "_ => eval("'42'")" did not throw
[FAIL] Blocked indirect eval in DedicatedWorkerGlobalScope.
  assert_throws_js: function "_ => eval?.("'42'")" did not throw
[FAIL] Blocked Function constructor in DedicatedWorkerGlobalScope.
  assert_throws_js: function "_ => new functionConstructor("return;")" did not throw
[FAIL] Blocked AsyncFunction constructor in DedicatedWorkerGlobalScope.
  assert_throws_js: function "_ => new functionConstructor("return;")" did not throw
[FAIL] Blocked GeneratorFunction constructor in DedicatedWorkerGlobalScope.
  assert_throws_js: function "_ => new functionConstructor("return;")" did not throw
[FAIL] Blocked AsyncGeneratorFunction constructor in DedicatedWorkerGlobalScope.
  assert_throws_js: function "_ => new functionConstructor("return;")" did not throw
Harness: the test ran to completion.

